src/scipy_api_list.py

```diff
@@ -0,0 +1,77 @@
+"""
+
+This is a python script to show API for each submodules of SciPy
+
+"""
+import importlib
+import inspect
+from collections import defaultdict
+
+import numpy
```

Comment:
[Question] Why we need to import numpy? Just checking numpy version?

Reply:
> Just checking numpy version?

Yes, the current`print_scipy_api_list.py` in master branch still privides the version of numpy.

https://github.com/AtsushiSakai/SciPy.jl/blob/ddf4eab371cf47ee0a9550edd69a2d1c6770974f/src/print_scipy_api_list.py#L7

Do you use this functionality? If not, I'll remove lines depend on numpy e.g. `import numpy` and `numpy.__version__` etc...

Reply:
Ah, I forget it. Maybe  this is debug use when scipy has an issue because of numbly version. Let’s keep it